roles/security/tasks/suricata.yml

```diff
@@ -71,10 +69,33 @@
   - emerging-inappropriate
   - emerging-imap
   - emerging-voip
+  - tor
+  - emerging-p2p
+  - emerging-web_specific_apps
+  - emerging-activex
+  - emerging-web_server
+  register: suri_cfg
+  tags:
+  - security
+  - ids
+
+- name: suricata | Disable specific IDS alerts
+  lineinfile: dest=/etc/suricata/rules/{{ item.rules }}.rules
+              regexp='^(alert .*{{ item.alert }}.*)$'
+              line='#\1'
+              backrefs=yes
+              backup=yes
+  with_items:
+    - { rules: 'emerging-policy', alert: 'YUM User-Agent' }
```

Comment:
Yeah, definitely, that sounds like the way to go. I was thinking about making them rule arguments, but being host variables would make it cleaner :)